frontend/app/features/register/controllers/registerController.js

```diff
@@ -3,26 +3,43 @@
 
 
     angular.module('em.register').controller('em.register.registerController', registerController);
-    registerController.$inject = ['UserService', '$location', '$rootScope', 'FlashService'];
+    registerController.$inject = ['$scope', 'userService', '$location', '$rootScope', 'flashService'];
 
-    function registerController(UserService, $location, $rootScope, FlashService) {
-        var vm = this;
+    function registerController($scope, userService, $location, $rootScope, flashService) {
 
-        vm.register = register;
+        $scope.register = register;
```

Comment:
To make core more readable you could declare you functions immediately
`$scope.register = function(){}`